spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/Topology.scala

```diff
@@ -154,33 +137,37 @@ class Sequential[T: ClassTag](val stopInferShape: Boolean = false)
    * @param module module to be add
    * @return this container
    */
-  override def add(module: AbstractModule[_ <: Activity, _ <: Activity, T]): this.type = {
+  def add(module: AbstractModule[_ <: Activity, _ <: Activity, T]): this.type = {
     if (frozen) {
       throw new RuntimeException(
         "This Sequential has been frozen, as it has been added into other container")
     }
     if (module.isInstanceOf[Sequential[T]]) {
       module.asInstanceOf[Sequential[T]].frozen = true
     }
-    Util.excludeNotKeras[T](Seq(module))
-    if (!stopInferShape) {
-      triggerBuilding(module)
-    }
-    modules += module.asInstanceOf[AbstractModule[Activity, Activity, T]]
+    validateInput[T](Seq(module))
```

Comment:
validateInput is not correct for `add`

Reply:
fixed in: https://github.com/intel-analytics/BigDL/pull/2293/files#diff-887accbfc37525584b59f19718396065R97